client/src/redux/ga-saga.js

```diff
@@ -2,7 +2,7 @@ import { all, call, takeEvery } from 'redux-saga/effects';
 import TagManager from '../analytics';
 
 function* callGaType({
-  payload: { action, duration, amount, event, pagePath }
+  payload: { action, duration, amount, event, pagePath, render_time_msec }
```

Comment:
```suggestion
  payload: { action, duration, amount, event, pagePath, challengeRenderTime }
```
Passing around snake cased variables is a little ugly.